.changeset/thirty-flowers-move.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I get why the tooling suggested all these packages, but in this case we should override because the only place where we're actually changing the public API surface of a package is telemetry-utils. That should be the only one listed here, with a `major` kind of change.

The title could be something like "Internal telemetry types removed from public API surface", and the description should list the specific types for which we changed the tag, and what consumers of FluidFramework need to do in case they were using them, which mostly should be `Use this "base" type instead` (e.g. `ITelemetryPerformanceEventExt` -> `ITelemetryBaseEvent` from @core-interfaces ).

Ping me if you have issues figuring out which base types are the "replacements" for each of these.